### PR TITLE
[Messenger] Add tests for ValidationMiddleware

### PR DESCRIPTION
**Pull request by yceruto**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Component/Messenger/Tests/Middleware/ValidationMiddlewareTest.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            ->willReturn($this->createMock(ConstraintViolationListInterface::class))
+        ;
+        $next = $this->createPartialMock(\stdClass::class, array('__invoke'));
+        $next
+            ->expects($this->once())
+            ->method('__invoke')
+            ->with($message)
+        ;
+
+        (new ValidationMiddleware($validator))->handle($message, $next);
```
Comment (sroze) [inline src/Symfony/Component/Messenger/Tests/Middleware/ValidationMiddlewareTest.php]: Could you also test the return value?

**Review by sroze — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Messenger/Tests/Middleware/ValidationMiddlewareTest.php (hunk 1 above)
Comment (yceruto) [reply src/Symfony/Component/Messenger/Tests/Middleware/ValidationMiddlewareTest.php]: Done.

**Comment by yceruto** [comment]
Status: Needs Review

**Review by ogizanagi — APPROVED** [review]

File: src/Symfony/Component/Messenger/composer.json [hunk 2]
```diff
@@ -25,7 +25,8 @@
         "symfony/property-access": "~3.4|~4.0",
         "symfony/var-dumper": "~3.4|~4.0",
         "symfony/property-access": "~3.4|~4.0",
-        "symfony/process": "~4.0"
+        "symfony/process": "~4.0",
+        "symfony/validator": "~4.0"
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Messenger/composer.json]: would 3.4 work (same for process?)
can you fix the alpha order also? (this is a new component so it's still time to fix this)

**Review by nicolas-grekas — APPROVED** [review]
with minor comment

**Comment by nicolas-grekas** [comment]
Thank you @yceruto.

**Pull request merged**